From 069bb25961377cc2fb8056405d67196505234655 Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Wed, 1 Jul 2020 17:09:33 +0800
Subject: [PATCH] ARM: dts: rv1126: move the tsadc parameters

This patch moves the tsadc default parameters to the rv1126.dtsi.
When the hardware changes, the tsadc parameters need to be
reconfigured in DTS.(Rockchip_Developer_Guide_Thermal_CN 3.2)

Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
Change-Id: I6ff427e703006135e32e5748e89b3a1367a035c9
---
 arch/arm/boot/dts/rv1126.dtsi         | 10 ++++++++++
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 10 ----------
 2 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 7b6fbdac8ef9..795570f9bad2 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -1350,6 +1350,11 @@
 		#thermal-sensor-cells = <1>;
 		nvmem-cells = <&cpu_tsadc_trim_l>, <&cpu_tsadc_trim_h>, <&tsadc_trim_base>;
 		nvmem-cell-names = "trim_l", "trim_h", "trim_base";
+		rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
+		rockchip,hw-tshut-polarity = <0>; /* tshut polarity 0:LOW 1:HIGH */
+		pinctrl-names = "gpio", "otpout";
+		pinctrl-0 = <&tsadcm0_pins>;
+		pinctrl-1 = <&tsadc_shutorg>;
 		status = "disabled";
 	};
 
@@ -1370,6 +1375,11 @@
 		#thermal-sensor-cells = <1>;
 		nvmem-cells = <&npu_tsadc_trim_l>, <&npu_tsadc_trim_h>, <&tsadc_trim_base>;
 		nvmem-cell-names = "trim_l", "trim_h", "trim_base";
+		rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
+		rockchip,hw-tshut-polarity = <0>; /* tshut polarity 0:LOW 1:HIGH */
+		pinctrl-names = "gpio", "otpout";
+		pinctrl-0 = <&tsadcm0_pins>;
+		pinctrl-1 = <&tsadc_shutorg>;
 		status = "disabled";
 	};
 
diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index 2882c7b50222..29bc54da3424 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -213,11 +213,6 @@
 };
 
 &cpu_tsadc {
-	rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
-	rockchip,hw-tshut-polarity = <0>; /* tshut polarity 0:LOW 1:HIGH */
-	pinctrl-names = "gpio", "otpout";
-	pinctrl-0 = <&tsadcm0_pins>;
-	pinctrl-1 = <&tsadc_shutorg>;
 	status = "okay";
 };
 
@@ -1024,11 +1019,6 @@
 };
 
 &npu_tsadc {
-	rockchip,hw-tshut-mode = <0>; /* tshut mode 0:CRU 1:GPIO */
-	rockchip,hw-tshut-polarity = <0>; /* tshut polarity 0:LOW 1:HIGH */
-	pinctrl-names = "gpio", "otpout";
-	pinctrl-0 = <&tsadcm0_pins>;
-	pinctrl-1 = <&tsadc_shutorg>;
 	status = "okay";
 };
 
-- 
2.35.3

